src/TextForEvent.js

```diff
@@ -75,8 +75,8 @@ function textForMemberEvent(ev) {
                 } else if (!prevContent.avatar_url && content.avatar_url) {
                     return _t('%(senderName)s set a profile picture.', {senderName});
                 } else {
-                    // suppress null rejoins
-                    return '';
+                    // This is a null rejoin, it will only be visible if the Labs option is enabled
```

Comment:
Hmm, can you point to the code that will hide these without the labs option? Maybe I am missing something... There's a check in `EventTile.haveTileForEvent`, but it won't be filtering these out any more:

https://github.com/matrix-org/matrix-react-sdk/blob/1c7af38d83a6d3880f92db4e2af8924cf6b4215a/src/components/views/rooms/EventTile.js#L836-L837

Reply:
True, I have put the SettingsStore check in, and whilst I know it is an "expensive" call, here in theory it is super-rare as it is only called on null-rejoins and passing it through seems untoward due to it being a very specific thing.